Shared/Features/Freaks/View/FreaksView.swift

```diff
@@ -3,11 +3,28 @@ import SwiftUI
 struct FreaksView: View {
     var viewModel = FreaksViewModel()
 
+    init() {
+        let coloredNavAppearance = UINavigationBarAppearance()
+
```

Comment:
Minor: we can get rid of extra lines between initialization and setup for the same object.